client/src/featureform/register.py

```diff
@@ -983,6 +998,45 @@ def get_redis(self, name):
         fakeProvider = Provider(name=name, function="ONLINE", description="", team="", config=fakeConfig)
         return OnlineProvider(self, fakeProvider)
 
+    def get_azure_blob(self, name):
```

Comment:
Should this be get_blob_store instead so it matches the register_blob_store call?

Reply:
yes